src/bot.py

```diff
@@ -13,10 +13,10 @@
     format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO
 )
 
-logger = logging.getLogger(__name__)
+logger: logging.Logger = logging.getLogger(__name__)
 
 
-def give_date():
+def give_date() -> str:
     today = datetime.date.today()
     day, year = today.day, today.year
     month = str(today).split('-')[1]
```

Comment:
There are missing annotations for these variables, `today, day, year, month`

Reply:
I was looking for the types of datetime but I got confused. 
I took deeper look to the corresponding class so I  am going to annotate as ```today : datetime.date```, ```day: datetime.date.day``` etc. and test if I get errors.